src/test/service/filterServiceSpec.js

```diff
@@ -0,0 +1,10 @@
+var filterService = require('../../service/filterService')
+var configUtil = require('../../libs/sb-config-util')
+
+describe('filter service', function () {
+  it('check for getChannelSearchString method', function () {
+    spyOn(configUtil, 'getConfig')
+    filterService.getChannelSearchString(function (obj) {})
+    expect(configUtil.getConfig).toHaveBeenCalled()
```

Comment:
this is an insufficient expectation, also need to check that the correct filters have been added to the query.

Reply:
we are testing the value in the requestMiddlewareSpec

Reply:
in this test we should use `configUtil.setConfig` before the call and then add an expectation on the return value of `configUtil.getConfig`